client/scss/components/_find_us.scss

```diff
@@ -0,0 +1,39 @@
+//* @define find-us;
+
+.find-us {
+  @include font('HNL5');
+  border-top: 1px solid color('charcoal');
+  padding-top: 1.6rem;
+  color: color('white');
+
+  .icon {
+    display: block;
+    float: left;
+    width: 14px;
```

Comment:
I think you're right, I'll change to em in this instance.

RE: default - not all the icons are the same aspect ratio, so if we have a default, some will end up with extra space around them. I need to check, but if most of them are the same, that could be the default and then we would only have to override occasionally